### PR TITLE
Adding a New User to Market Application

### PR DESCRIPTION
**Pull request by onurd86**
Add a new user to market app. It utilizes BCrypt Encoder for storing passwords. So, it was just adding a new entry to the database script.

### CODE REVIEW

File: jdk_11_maven/em/embedded/rest/market/src/main/java/em/embedded/market/EmbeddedEvoMasterController.java [hunk 1]
```diff
@@ -138,7 +138,8 @@ public SutInfoDto.OutputFormat getPreferredOutputFormat() {
     public List<AuthenticationDto> getInfoForAuthentication() {
```
Comment (arcuri82) [inline jdk_11_maven/em/embedded/rest/market/src/main/java/em/embedded/market/EmbeddedEvoMasterController.java:138]: need to update `ExternalEvoMasterController` as well

**Review by arcuri82 — CHANGES_REQUESTED** [review]

File: jdk_11_maven/em/embedded/rest/market/src/main/java/em/embedded/market/EmbeddedEvoMasterController.java (hunk 1 above)
Comment (arcuri82) [reply jdk_11_maven/em/embedded/rest/market/src/main/java/em/embedded/market/EmbeddedEvoMasterController.java:138]: @onurd86  btw, I assume you then run EvoMaster on this update settings, and verified there was no message saying 401 with this new user, isn't it?

Comment (onurd86) [reply jdk_11_maven/em/embedded/rest/market/src/main/java/em/embedded/market/EmbeddedEvoMasterController.java:138]: Hi Andrea,

Thanks for mentioning that. I tried to run EvoMaster again after adding the new user. I do not get any 401 after adding the new user. Adding a new user to market was very easy.

Comment (onurd86) [reply jdk_11_maven/em/embedded/rest/market/src/main/java/em/embedded/market/EmbeddedEvoMasterController.java:138]: I have updated External Controller as well.

Comment (onurd86) [reply jdk_11_maven/em/embedded/rest/market/src/main/java/em/embedded/market/EmbeddedEvoMasterController.java:138]: This was one of the successful tests generated by EvoMaster after adding user2:

 ```
@Test @Timeout(60)
    public void test_1() throws Exception {
        
        given().accept("*/*")
                .header("Authorization", "Basic dXNlcjJAeWFuZGV4LnJ1Onl1cmlkb2xnb3J1a2k=") // user2
                .header("x-EMextraHeader123", "")
                .get(baseUrlOfSut + "/customer/orders?name=ATX")
                .then()
                .statusCode(200)
                .assertThat()
                .contentType("application/json")
                .body("size()", equalTo(0));
    }
```

**Review by arcuri82 — APPROVED** [review]

**Pull request merged**